src/lib/storageProvider.ts

```diff
@@ -4,6 +4,17 @@ export interface StorageProviderInfo {
   sectorSize: string;
   windowPostPartitionSectors: string;
   windowPostProofType: string;
+  dealParams: DealParams;
+}
+
+interface DealDuration {
```

Comment:
Ideally, this should also be exported since its indirectly exported by the `DealParams`